project/OmnidocBuild.scala

```diff
@@ -18,6 +18,7 @@ object OmnidocBuild extends Build {
   val scalaTestPlusPlayVersion = sys.props.getOrElse("scalatestplus-play.version", "2.0.0-M2")
   val anormVersion             = sys.props.getOrElse("anorm.version",              "2.6.0-M1")
   val playEbeanVersion         = sys.props.getOrElse("play-ebean.version",         "4.0.0-M1")
+  val playJsonVersion          = sys.props.getOrElse("play-json.version",          "2.6.0-M3")
```

Comment:
You mean another version of Play JSON that actually includes the new docs? Yes, we should. :)